packages/flutter_tools/lib/src/commands/init.dart

```diff
@@ -129,9 +145,8 @@ const String _pubspec = r'''
 name: {{projectName}}
 description: {{description}}
 dependencies:
-  flutter: ">=0.0.2 <0.1.0"
-dev_dependencies:
-  sky_tools: any
+  flutter:
+    path: {{flutterPackagePath}}
```

Comment:
It's a bit hard to get my head around writing a path dep into the pubspec, but I understand the argument for avoiding version skew for the user.